user/deployment-v2.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Deployment (v2)
+layout: en
+---
+
+> This page documents deployments using the next major version dpl v2 which
+> currently is in a preview release phase. Please see our blog post on details
+> about the release process. Documentation for dpl v1, the current default
+> version, can be found [here](/user/deployments).
+
+## Supported Providers
+
+Continuous Deployment to the following providers is supported:
+
+{% include deployments.html %}
+
+To deploy to a custom or unsupported provider, use the [after-success build
+stage](/user/deployment/custom/) or [script provider](/user/deployment/script).
```

Comment:
```suggestion
step](/user/deployment/custom/) or [script provider](/user/deployment/script).
```
consistency wise, I believe stages are the ones people can define through build stages while the different sections that run during the build life-cycle are steps